laravel/resources/js/admin.js

```diff
@@ -88,6 +98,21 @@ $(document).ready(function()
     {
         $('.user-role').value($('.user-role').data('role'));
         $('.buttons').attr('style', false);
+
+        //pull data array from earlier and assign the values based on prevData
+        $('.user-roles input').each(function(self)
+        {
+            self = $(this);
```

Comment:
For consistency with the rest of the project, you should just use `$(this)` instead of redefining it